src/Libs/cairo-1.0/Records/Surface.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Cairo
+{
+    public partial class Surface
+    {
+        public Content Content => Internal.Surface.GetContent(Handle);
+        public Device Device => new Device(Internal.Surface.GetDevice(Handle));
```

Comment:
If the retrieved handle of `Internal.Surface.GetDevice(Handle)` never changes we could cache the created instance.

If it seldomly changes we could cache it, too and compare if the retrieved handle is different to our the cached handle. If it is different we create a new instance if not, we return the cached instance.

Otherwise this should probably not be a property but a method as it is not expected of properties to return new references on each call.

Reply:
I don't see any methods for changing it on a surface instance, so unless I'm missing something I think the native instance could be cached.